index.html

```diff
@@ -2540,18 +2605,438 @@ <h1>
+<a href="#generic-did-url-parameters"></a>) insofar as they are not part of
+the <var>input <a>DID</a></var> or <var>input <a>DID URL</a></var> syntax.
+Resolver options can for example be used to control caching or content negotiation.
+      </p>
+
+      <p>
+<a>Resolver options</a> are expressed as a map of name-value string pairs.
+      </p>
+      <p>
+Common fields are included here. Additional <a>resolver options</a> MAY be method-independent or method-specific.
+      </p>
+      <p>
+The concrete representation of <a>resolver options</a> MUST be defined by the applicable <a>binding</a> of
+the <a>DID resolution</a> or <a>DID URL dereferencing</a> function.
+      </p>
+
+      <p class="note">
+For example, a concrete <a>binding</a> of the <a>DID resolution</a> or <a>DID URL
+dereferencing</a> function might define how resolver options are represented in the form
+of headers of a network protocol.
+      </p>
+
+      <p>
+This specification defines a single concrete <a>resolver option</a>:
+      </p>
+      <ul>
+        <li>
+The <code>accept</code> <var>resolver option</var> can be used to indicate the requested
+MIME type of the representation of the resource that is returned by the <a>DID resolution</a>
+or <a>DID URL dereferencing</a> function.
+          <ul>
+            <li>
+This <var>resolver option</var> is OPTIONAL.
+            </li>
+            <li>
+If this <var>resolver option</var> is supplied to the <a>DID resolution</a> function,
+or if it is supplied to the <a>DID URL dereferencing</a> function and the result is expected
+to be a <a>DID document</a>, then the value of this <var>resolver option</var> MUST be a
+MIME type associated with a compliant representation of the abstract data model for
+<a>DID documents</a> (see Section <a href="#representations"></a>).
+            </li>
+          </ul>
+        </li>
+      </ul>
+
+      <p>
+A <a>DID resolver</a> MAY support additional <var>resolver options</var>.
+      </p>
+
+    </section>
+
+    <section>
+      <h2>
+Metadata
+      </h2>
+
+      <p>
+In addition to <a>DID documents</a>, the <a>DID resolution</a> and <a>DID URL dereferencing</a> functions
+return <a>DID document metadata</a> as well as <a>DID resolution metadata</a>.
+      </p>
+      <p>
+These sets of metadata are expressed as maps of name-value string pairs, referred
+to as <var>metadata properties</var>. The keys of the map are unique and MUST NOT
+be repeated.
+      </p>
+      <p>
+Common fields are included here. Additional <var>metadata properties</var> MAY be
+method-independent or method-specific.
+      </p>
+      <p>
+While the concrete representation of metadata MAY be defined by the applicable <a>binding</a> of
+the <a>DID resolution</a> or <a>DID URL dereferencing</a> function, the metadata
+MUST be made available to the caller of the <a>DID resolution</a> or <a>DID URL dereferencing</a>
+function as a map of associated plain string keys and values. 
```

Comment:
"plain string keys and values" => "key-value pairs". In the case of CBOR they wouldn't have strings and I think it could be confusing when considering the statements above about returning in a compliant format.

Reply:
The point is that the format on the wire is irrelevant. It would be returned to the client calling the function as a string and input as a string, or equivalent, regardless of how it's encoded underneath. You shouldn't have to know that CBOR tag "2" means "content-type", for example. The goal is to get away from the constraints and capabilities of different representations for the metadata bits. #253 tries to make this more explicit.